src/components/Tasks/TaskListItem.tsx

```diff
@@ -57,7 +57,8 @@ export default React.memo(function TaskListItem(props: PropsType) {
   return (
     <ListItem
       primaryText={title}
-      secondaryText={task.dueDate && `Due ${moment().to(task.dueDate.toJSDate())}`}
+      secondaryText={task.dueDate && `Due ${task.dueDate.isDate && task.dueToday ? 'today' : moment(task.dueDate.toJSDate()).fromNow()}`}
```

Comment:
1. This change is unrelated to the PR, please don't shove in unrelated changes.
2. This is not desirable behaviour. The problem is that this doesn't update every time the clock ticks, so it gets stale very quickly... That's generally the problem with `fromNow`... To solve it, you would need to create a widget that automatically updates as the clock ticks (with configurable ticking resolution). It's not hard, just pointless for now...

Reply:
So should we just not track due date at all in the UI?

Reply:
No, due date is fine, and should be shown, the problem is only with "fromNow". The moment you show the actual date, it's fine.

Reply:
Ok, I can change it to just show the due date. I just want to point out thought that the old code had already been showing the time to the due date. This `moment().to(task.dueDate.toJSDate())` is the same as this `moment(task.dueDate.toJSDate()).fromNow()`. 

I can get rid of that though, if you'd like.

Reply:
Oh, that was a mistake most likely. Yeah, remove the `.to`, just show the time.

Reply:
Ok, I will revert this change for this PR, and then fix that in a separate PR.